Makefile

```diff
@@ -717,6 +716,12 @@ git-gaia-node-modules: gaia_node_modules.revision
 	fi
 	(cd "$(NODE_MODULES_SRC)" && git fetch && git reset --hard "$(NODE_MODULES_REV)" )
 
+npm-cache:
```

Comment:
can we extract this into a shell script and invoke it as part of npm install (or as a post hook if needed)